### PR TITLE
Fixed #24978 -- Escape special characters in fixture paths

### PR DESCRIPTION
**Pull request by MoritzS**
Ticket: https://code.djangoproject.com/ticket/24978

### CODE REVIEW

File: django/utils/six.py [hunk 1]
```diff
@@ -852,3 +854,34 @@ def python_2_unicode_compatible(klass):
     else:
         memoryview = buffer
     buffer_types = (bytearray, memoryview)
+
+
+# backport of Python 3.4's glob.escape
```
Comment (timgraham) [inline django/utils/six.py]: I'd rather put this in a separate module like `django.utils.glob` unless you plan to try to get it included in six itself.

File: docs/releases/1.9.txt [hunk 2]
```diff
@@ -270,6 +270,9 @@ Management Commands
 * The :djadmin:`createcachetable` command now has a ``--dry-run`` flag to
   print out the SQL rather than execute it.
 
+* The :djadmin:`loaddata` command now works correctly with special characters
```
Comment (timgraham) [inline docs/releases/1.9.txt]: We don't typically list bug fixes in the major release notes.

File: tests/fixtures/tests.py [hunk 3]
```diff
@@ -224,6 +224,11 @@ def test_dumpdata_with_excludes(self):
                 "Unknown model in excludes: fixtures.FooModel"):
             self._dumpdata_assert(['fixtures', 'sites'], '', exclude_list=['fixtures.FooModel'])
 
+    def test_load_fixture_with_special_characters(self):
+        Site.objects.all().delete()
```
Comment (timgraham) [inline tests/fixtures/tests.py]: Would be slightly better to use non-contrib models in case sites is someday removed.

**Comment by timgraham** [comment]
commit msg: "Escaped" for past tense.

File: django/utils/six.py (hunk 1 above)
Comment (MoritzS) [inline django/utils/six.py]: I just didn't want to create a new file for that, but I guess it's better that way.

**Comment by timgraham** [comment]
merged in 98df288ddaba9787e4a370f12aba51c2b9133142, thanks!

**Pull request closed without merging**